pkg/cmd/util/clientcmd/factory.go

```diff
@@ -122,7 +122,7 @@ func (c defaultingClientConfig) ClientConfig() (*kclient.Config, error) {
 		return icc, nil
 	}
 
-	return nil, fmt.Errorf(`No configuration file found, please login or point to an existing file:
+	return nil, fmt.Errorf(`No (or empty) configuration file found, please login or point to an existing file:
```

Comment:
that reads very strangely. pretty sure there are tests that need updating as well